pkg/driver/request_coalescing_test.go

```diff
@@ -175,13 +172,14 @@ func testRequestFail(t *testing.T, executor modifyVolumeExecutor) {
 		return 0, fmt.Errorf("ResizeOrModifyDisk failed")
 	})
 
+	options := &Options{
+		ModifyVolumeRequestHandlerTimeout: 2 * time.Second,
```

Comment:
Top level comment: `TestVolumeModificationWithCoalescing` subtests can take up to 7 seconds to run at worst, hinting that we are sleeping for too long in unit tests. Can we minimize the sleeps in tests? 

```
TestVolumeModificationWithCoalescing/basic_request_coalescing_success:_external-resizer,2
TestVolumeModificationWithCoalescing/basic_request_coalescing_success:_volume-modifier-for-k8s,2
TestVolumeModificationWithCoalescing/duplicate_requests:_external-resizer,2
TestVolumeModificationWithCoalescing/duplicate_requests:_volume-modifier-for-k8s,2
TestVolumeModificationWithCoalescing/partial_fail:_external-resizer,2
TestVolumeModificationWithCoalescing/partial_fail:_volume-modifier-for-k8s,2
TestVolumeModificationWithCoalescing/request_fail:_external-resizer,2
TestVolumeModificationWithCoalescing/request_fail:_volume-modifier-for-k8s,2
TestVolumeModificationWithCoalescing/sequential_requests:_external-resizer,7
TestVolumeModificationWithCoalescing/sequential_requests:_volume-modifier-for-k8s,7
TestVolumeModificationWithCoalescing/timing:_external-resizer,7
TestVolumeModificationWithCoalescing/timing:_volume-modifier-for-k8s,7
```

Reply:
See the PR description: this PR doesn't make any significant modifications to the tests on purpose.